test/e2e/apimachinery/chunking.go

```diff
@@ -75,7 +75,12 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		})
 	})
 
-	ginkgo.It("should return chunks of results for list calls", func() {
+	/*
+		Release : v1.15
+		Testname: Return chunks of results for large numbers of objects
+		Description: List a large number of objects and verify the first three chunks
+	*/
+	framework.ConformanceIt("should return chunks of results for list calls", func() {
```

Comment:
This description needs a lot of work imo.

Reply:
So in looking through the test it's only exercising pod templates and if were going to evaluate behavior I'd argue this test needs to a lot more work.

Reply:
@bobymcbobs can you update with the RFC2119 style description?

@timothysc We need to exercise something to list in order to return chunks, pod templates happens to be what is used here. We could shift to listing something else if something might fit better. It does happen to use an endpoint that no current conformance test hits, hence the prioritization as a possible quick promotion.

Reply:
So you could use the dynamic client to hit chunk across multiple resource kinds.  Just promoting the existing test that is poorly named is not sufficient.  1. Make a robust test that exercises a behavior, 2. promote.  I'd argue we this test needs #1 first.

Reply:
The idea of #78748 was to promote existing tests, add metadata and updating descriptions when those tests already hit endpoints that increase coverage.

If this test isn't robust enough, let's go ahead and close it.

I'd rather prioritize writing new tests based on stable endpoints hit by core kubernetes components:
https://github.com/kubernetes/kubernetes/issues/78749.

Reply:
We are generally not interested in increasing raw endpoint coverage indirectly, or through poorly written/phrased tests.  We want to be clear and that we are exercising expected behavior that we want to be consistent across providers in a deliberate and well thought out manner. 

/cc @spiffxp

Reply:
@timothysc can you clarify whether you believe this test is not a good test for the behavior it is setting out to exercise, or if you are saying that it only indirectly covers the endpoint in question (`createCoreV1NamespacedPodTemplate`) hence it should not be considered a good test for *that specific endpoint*. If it is the former, then we will close this PR promptly. If it's the latter, then we could continue to work on promoting this test. Apologies if I'm misunderstanding the comments you've already made, but this is the question which still remains for me.

Reply:
In looking at the issue that motivated opening this PR, if our goal is to cover `createCoreV1NamespacedPodTemplate` this isn't the test to do it.  It _happens_ to do it, but a more straightforward CRUD test would be more appropriate.